lms/djangoapps/instructor/enrollment.py

```diff
@@ -244,6 +245,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
                     requesting_user_id=requesting_user_id
                 )
                 submission_cleared = True
+                SCORE_CHANGED.send(
```

Comment:
What's the rationale for calling it from within this clause?  Why not fire the signal towards the end of this method within `if delete_module:` and after `module_to_reset.delete()` is called?  This way, when we rescore the problem, all state changes have completed.